src/main/java/it/gov/pagopa/apiconfig/selfcareintegration/controller/CreditorInstitutionController.java

```diff
@@ -255,4 +268,29 @@ public ResponseEntity<CIAssociatedCodeList> getSegregationCodesFromCreditorInsti
         creditorInstitutionsService.getSegregationCodesFromCreditorInstitution(
             creditorInstitutionCode, showUsedCodes, service));
   }
+
+    /**
+     * Retrieve a list of creditor institution business names, given the provided list of creditor institution tax codes
+     *
+     * @param taxCodeList the list of creditor institution tax codes
+     * @return a list of {@link CreditorInstitutionInfo}, containing the business name and tax code of creditor institution
+     */
+    @Operation(summary = "Get the list of creditor institution business names",
+            description = "Return a list of business name and tax code of creditor institutions, given the provided list of creditor institution tax codes",
+            security = {@SecurityRequirement(name = "ApiKey"), @SecurityRequirement(name = "Authorization")})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            array = @ArraySchema(schema = @Schema(implementation = CreditorInstitutionInfo.class)))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "404", description = "Not Found",
+                    content = @Content(schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "500", description = "Service unavailable",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
+    @PostMapping(value = "", produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
```suggestion
    @GetMapping(value = "", produces = {MediaType.APPLICATION_JSON_VALUE})
```

Reply:
it's a POST because it has a request body with the list of tax code. You prefer GET with a body?